lib/rubocop/cop/lint/boolean_symbol.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for boolean symbols.
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #   :true
+      #
+      # @example
+      #
+      #   # good
+      #
+      #   true
+      #
+      class BooleanSymbol < Cop
+        MSG = 'Boolean symbol - you probably meant to use `%s`.'.freeze
```

Comment:
Symbol with a boolean value name (or something like this). "Boolean symbol" sounds a bit strange to me.